UI/src/view/Layer.js

```diff
@@ -47,10 +47,10 @@
     this.handleMapMoved = function(state) {
       if (state.selectedLayer === layerName && state.zoom >= me.minZoomForContent) {
         if (!me.isStarted()) {
-          me.start();
+          me.start('moved');
```

Comment:
Noone seems to be using this parameter for anything so perhaps it can be removed